WordPressKit.xcodeproj/project.pbxproj

```diff
@@ -675,6 +678,10 @@
 
 /* Begin PBXFileReference section */
 		0152100B28EDA9E400DD6783 /* StatsAnnualAndMostPopularTimeInsightDecodingTests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = StatsAnnualAndMostPopularTimeInsightDecodingTests.swift; sourceTree = "<group>"; };
+		0C3A2A412A2E7BA500FD91D6 /* CHANGELOG.md */ = {isa = PBXFileReference; lastKnownFileType = net.daringfireball.markdown; path = CHANGELOG.md; sourceTree = "<group>"; };
```

Comment:
I added the CHANGELOG.md file to Xcode project so that it's easier to modify directly from the project.